calico_kubernetes/calico_kubernetes.py

```diff
@@ -724,6 +723,84 @@ def _log_interfaces(namespace):
         # Don't exit if we hit an error logging out the interfaces.
         logger.exception("Ignoring error logging interfaces")
 
+
+def load_config():
+    """
+    Loads configuration for the plugin - returns a dictionary.
+
+    Looks first in environment, then in local config file.
+    """
+    # First, read the config file and get defaults.
+    config = read_config_file()
+
+    # Get config from environment.
+    env_config = {
+            CALICO_IPAM_VAR: os.environ.get(CALICO_IPAM_VAR),
```

Comment:
Could save code with:

```
for var in [CALICO_IPAM_VAR, ...]:
    config[var] = os.environ.get(var, config[var])
```

? Could even pull the ENV_VARS list out as a constant if you want to keep this function tidy.

Or could even make this more generic with

```
for var in config:
    os.environ.get(var, config[var])
```

Not sure if I prefer this, since it allows you to load any env variable you want into the plugin...

The existing code works, but I find this sort of repeated block to be error-prone (easy to fail to update one of the calls down the road, for example).